### ISSUE TITLE
Change ref-based RNA-seq tutorial to use fastqsanger.gz instead of fastqsanger files

### ISSUE THREAD

**Issue by mblue9** [galaxyproject/training-material#898]
In the ref-based RNA-seq tutorial (here: http://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html#data-upload) uncompressed fastq files are currently used as input but it would be better to use compressed fastqs because:
1) it would save on storage/quotas
2) compressed files are more common in real-world analyses

Requires replacing the fastqsanger files in Zenodo (here: https://zenodo.org/record/1185122) with fastqsanger.gz versions.

**Comment by mblue9** [comment]
I'm currently testing loading in the 4 example fastqsanger files from the tutorial to usegalaxy.org and  it's been 20 mins so far and they're still uploading. That seems quite slow (too slow?) to me, seems like it would make usegalaxy.org not possible to use to run a training with this tutorial (for those steps anyway). Or will these files be made available in a data library within usegalaxy.org/.eu?

I'm loading the files using these links:
  https://zenodo.org/record/1185122/files/GSM461177_1.fastqsanger
  https://zenodo.org/record/1185122/files/GSM461177_2.fastqsanger
  https://zenodo.org/record/1185122/files/GSM461180_1.fastqsanger
  https://zenodo.org/record/1185122/files/GSM461180_2.fastqsanger

**Comment by mblue9** [comment]
The Trim Galore steps took 15 mins to run on those 4 fastqsanger files in usegalaxy.eu, which is right at the limit of the recommended max tool run time here: https://github.com/galaxyproject/training-material/pull/901. Wondering if using smaller files would be better?

**Comment by bgruening** [comment]
Its probably time to not replace trim-galore with fastp/cutadopt?

**Comment by mblue9** [comment]
Yep I'd already added that to the notes I'm making while reviewing this tutorial. Will add the notes to an issue when I'm done (things like the gtf version is also old - v87 from Dec 2016, current is v92 Apr 2018)

**Comment by mblue9** [comment]
@bgruening I don't know if something's just wrong today but those fastqsanger files are still uploading from Zenodo into usegalaxy.org for >2 hours now! (and that's the very first step of the tut)
